progress/react/icons/package.json

```diff
@@ -12,23 +12,34 @@
     },
     "prettier": "@pxblue/prettier-config",
     "peerDependencies": {
-        "@types/react": "^16.9.19",
-        "react": "^16.6.3"
+        "@material-ui/core": "^4.0.0",
+        "@material-ui/styles": "^4.5.0",
```

Comment:
@material-ui/core is a large dependency to add just to do styles...I would prefer if we don't depend on it in this package if we can help it. The styles package is smaller and might give us everything we need. Theres also react-jss which does css-in-js styles (https://cssinjs.org/?v=v10.4.0)

Reply:
I'm looking into this now; since a lot styles within these classes currently read from the props (size, color, etc) I'm planning to move these styles to be inline instead of inside of a class.  react-jss looks like it should support this transition with no problem.

Reply:
This worked pretty nicely.